cmd/config/subcommand/init/config_flags.go

```diff
@@ -0,0 +1,18 @@
+// Code generated by go generate; DO NOT EDIT.
+// This file was generated by robots.
+
+package init
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+)
+
+// GetPFlagSet will return strongly types pflags for all fields in Config and its nested types. The format of the
+// flags is json-name.json-sub-name... etc.
```

Comment:
Did you manually remove the generated functions ?

Reply:
yes